src/main/java/lotto/domain/Money.java

```diff
@@ -22,11 +27,24 @@ private void validateOverThousand(int inputMoney) {
 		}
 	}
 
-	public int divideThousand() {
-		return inputMoney / PRICE_ONE_LOTTO;
+	public boolean isBuyLotto(int lottoCount) {
```

Comment:
```suggestion
	public boolean canBuyLotto(int lottoCount) {
```

구매가 가능한지 확인하는 경우 can-으로 하는게 더 자연스러울 것 같아요!

Reply:
반환형이 boolean일땐 무조건 is를 써야돼! 라고 융통성 없이
생각했었습니다... 감사합니다!